source/conf.py

```diff
@@ -66,6 +63,7 @@
     "_extensions.post_process",
     "_extensions.rtd_patch",
     "_extensions.localization",
+    "_extensions.controls_js_sim.controls_js_build",
```

Comment:
Rename `controls_js_build.py` to `__init__.py`?
Then, this import would change to `_extensions.controls_js_sim`.
It would make it clearer which file has the setup.

Reply:
got it.